src/commcare_cloud/.bash_completion

```diff
@@ -1,50 +1,39 @@
-_envs()
-{
-    if [[ -z "$COMMCARE_CLOUD_ENVS" ]]
+_completion_props() {
+    if [[ -z "${COMPLETION_PROPS}" ]]
     then
-        COMMCARE_CLOUD_ENVS=$(commcare-cloud -h | head -n3 | tail -n1 | cut -d'{' -f2 | cut -d'}' -f1 | sed 's/,/ /g')
+        COMPLETION_PROPS="$(manage-commcare-cloud get environments commands ANSIBLE_DIR)"
     fi
-    echo $COMMCARE_CLOUD_ENVS
+    echo "$COMPLETION_PROPS"
 }
 
-_cmds()
-{
-    if [[ -z "$COMMCARE_CLOUD_CMDS" ]]
-    then
-        COMMCARE_CLOUD_CMDS=$(commcare-cloud -h | head -n4 | tail -n1 | cut -d'{' -f2 | cut -d'}' -f1 | sed 's/,/ /g')
-    fi
-    echo $COMMCARE_CLOUD_CMDS
-}
 
 _commcare_cloud()
 {
     local cur prev opts
     COMPREPLY=()
-    FILE_EXCHANGE_DIR=$(manage-commcare-cloud get-path ANSIBLE_DIR)
     cur="${COMP_WORDS[COMP_CWORD]}"
     prev="${COMP_WORDS[COMP_CWORD-1]}"
     opts="--help --version"
 
-    export COMMCARE_CLOUD_ENVS=$(_envs)
-    envs=$COMMCARE_CLOUD_ENVS
-
-    export COMMCARE_CLOUD_CMDS=$(_cmds)
-    cmds=$COMMCARE_CLOUD_CMDS
+    export COMPLETION_PROPS="$(_completion_props)"
+    COMMCARE_CLOUD_ENVS=$(echo "${COMPLETION_PROPS}" | head -n1 | tail -n1)
```

Comment:
True. I figured consistency with the lines below was nice, but I'll remove in case there's any callout overhead in invoking tail (guessing it's basically zero, but not worth wondering about :))